js/app.js

```diff
@@ -31,6 +31,217 @@ random number used for the computer's choice
 *********************/
 var randomNumber;
 
+/***********************
+The reset button
+*********************/
+var resetButton = document.querySelector('.reset');
+
+/***********************
+A variable to keep track if someone has won
+************************/
+var winner = false;
+
+/**********************
+keeps track of how many spots have been used
+************************/
+var imgCount = 0;
+
+/********************
+does this really need explaining
+*******************/
+var score = 0;
+
+/******************
+variable to display your score
+******************/
+var message = document.querySelector('.score');
+
+
+function checkForWinner() {
```

Comment:
Can make this DRY by converting to a simple for loop - You can likely simplify this by assigning a boolean value for each clicked box, creating an array of correct scores and iterating through that with the for loop.